### PR TITLE
SAM with receiver. Short declaration

### PR DESCRIPTION
**Pull request by turansky**

### CODE REVIEW

File: docs/topics/sam-with-receiver-plugin.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ does not have any built-in presets, and you need to specify your own list of spe
 
 ```groovy
 plugins {
-    id("org.jetbrains.kotlin.plugin.sam.with.receiver") version "$kotlin_version"
+    kotlin("plugin.sam.with.receiver") version "$kotlin_version"
```
Comment (koshachy) [inline docs/topics/sam-with-receiver-plugin.md:37]: Unfortunately, this code only works for Kotlin DSL, not Groovy.

You can improve your PR by adding tabs, similar to the [all-open plugin](https://github.com/JetBrains/kotlin-web-site/blob/master/docs/topics/all-open-plugin.md#gradle) documentation. Include the short declaration for Kotlin, but retain the existing example for Groovy.

**Review by koshachy — CHANGES_REQUESTED** [review]
Thank you for your contribution!

I've kindly requested some improvements to the PR, if that's okay with you.
Please feel free to ask any questions you might have.

File: docs/topics/sam-with-receiver-plugin.md (hunk 1 above)
Comment (turansky) [reply docs/topics/sam-with-receiver-plugin.md]: Done

Which version placeholder is preferred? With underscore or without?

**Review by koshachy — APPROVED** [review]
That was fast!
Thanks for the help, appreciate it!

File: docs/topics/sam-with-receiver-plugin.md (hunk 1 above)
Comment (koshachy) [reply docs/topics/sam-with-receiver-plugin.md]: Without,
Nice catch, there was a mistake in the code!

**Pull request merged**